src/atoms/Switch/Switch.tsx

```diff
@@ -7,8 +7,8 @@ import Typography from 'Typography';
 /* stylelint-disable unit-whitelist */
 /* stylelint-disable no-descending-specificity */
 const LabelText = styled(Typography)`
-  font-size: 0.875em;
   color: #163150;
+  margin-top: 12px;
```

Comment:
To keep sizes relative (so changing the browser's font size scales UI consistently), we should use em instead of rem. In this case `12 / 16 = 0.75` so we can replace `12px` with `0.75em`.

Reply:
Just pushed up a fix for this!